ui/retribution_paladin/presets.ts

```diff
@@ -67,37 +67,38 @@ export const Phase2RetSoMGear = PresetUtils.makePresetGear('P2 SoM', Phase2RetSo
 export const Phase3RetSoMGear = PresetUtils.makePresetGear('P3 SoM', Phase3RetSoMGearJson, {
 	customCondition: player => player.getLevel() == 50,
 });
-export const Phase4RetTwistGear = PresetUtils.makePresetGear('P4 Twist', Phase4RetGearJson, {
+export const Phase5ExodinGear = PresetUtils.makePresetGear('P5 Exodin', Phase5ExodinGearJson, {
 	customCondition: player => player.getLevel() == 60,
 });
-export const Phase4RetTwist6pT1Gear = PresetUtils.makePresetGear('P4 Ret Twist 6pT1', Phase4RetTwisting6PcT1GearJson, {
+export const Phase5SealStackingGear = PresetUtils.makePresetGear('P5 Seal Stacking', Phase5SealStackingGearJson, {
 	customCondition: player => player.getLevel() == 60,
 });
-export const Phase4RetExodinGear = PresetUtils.makePresetGear('P4 Ret Exodin', Phase4RetExodinGearJson, {
+export const Phase5ShockadinGear = PresetUtils.makePresetGear('P5 Shockadin', Phase5ShockadinGearJson, {
 	customCondition: player => player.getLevel() == 60,
 });
-export const Phase4RetExodin6pT1Gear = PresetUtils.makePresetGear('P4 Ret Exodin 6pT1', Phase4RetExodin6PcT1GearJson, {
+export const Phase5TwistingGear = PresetUtils.makePresetGear('P5 Twisting', Phase5TwistingGearJson, {
 	customCondition: player => player.getLevel() == 60,
 });
-export const Phase5TwistingGear = PresetUtils.makePresetGear('P5 Twisting', Phase5TwistingGearJson, {
+export const Phase6ExodinGear = PresetUtils.makePresetGear('P6 Exodin', Phase6ExodinGearJson, {
 	customCondition: player => player.getLevel() == 60,
 });
-export const Phase5ExodinGear = PresetUtils.makePresetGear('P5 Exodin', Phase5ExodinGearJson, {
+export const Phase6OneHandGear = PresetUtils.makePresetGear('P6 1H Ret', Phase6OneHandGearJson, {
 	customCondition: player => player.getLevel() == 60,
 });
-export const Phase5ShockadinGear = PresetUtils.makePresetGear('P5 Shockadin', Phase5ShockadinGearJson, {
+export const Phase6TwistingGear = PresetUtils.makePresetGear('P6 Twisting', Phase6TwistingGearJson, {
 	customCondition: player => player.getLevel() == 60,
 });
 
 export const GearPresets = {
 	[Phase.Phase1]: [Phase1RetGear],
 	[Phase.Phase2]: [Phase2RetSoCGear, Phase2RetSoMGear],
 	[Phase.Phase3]: [Phase3RetSoMGear],
-	[Phase.Phase4]: [Phase4RetTwistGear, Phase4RetTwist6pT1Gear, Phase4RetExodinGear, Phase4RetExodin6pT1Gear],
-	[Phase.Phase5]: [Phase5TwistingGear, Phase5ExodinGear, Phase5ShockadinGear],
+	[Phase.Phase4]: [],
```

Comment:
Looks like the Phase 4 sets are missing now?

Reply:
Yea the list was getting too big in the UI... feels like enough to just show current+previous phase just like Rogues did with APLs?
Sure, I can keep them here in the preset dict but showing 11 different preset gear options and 10 different APLs is... a bit much imo :D
especially when 4 of those are from *two* phases back

Reply:
I can add the relevant ones back just for completeness sake, but when P7 sets come, we will have to clean this up :)

Reply:
No that's fine totally get that